src/dss/GemJoinImplementationAbstract.sol

```diff
@@ -0,0 +1,17 @@
+pragma solidity >=0.5.12;
+
+// https://github.com/makerdao/dss/blob/master/src/join.sol
```

Comment:
LGTM, I would just change this to: https://github.com/makerdao/dss-deploy/blob/master/src/join.sol

Reply:
ah yeah good catch

Reply:
We will probably be moving all of the join adapters to dss, so this is fine.

Reply:
What was the reason to move all the join adapters to `dss`?
I'm not really sure is the best idea to overload `dss` with a bunch of different adapters. We might have several types in the future.